parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -749,5 +712,14 @@ pub mod pallet {
 			let signing_root = Self::compute_signing_root(header, domain)?;
 			Ok(signing_root)
 		}
+
+		pub fn latest_finalized_state() -> Result<CompactBeaconState, DispatchError> {
+			let latest_finalized_state =
+				match FinalizedBeaconState::<T>::get(LatestFinalizedBlockRoot::<T>::get()) {
+					Some(finalized_beacon_state) => finalized_beacon_state,
+					None => return Err(Error::<T>::NotBootstrapped.into()),
+				};
+			Ok(latest_finalized_state)
```

Comment:
We can shorten this with `ok_or`:
```suggestion
			FinalizedBeaconState::<T>::get(LatestFinalizedBlockRoot::<T>::get())
				.ok_or(Error::<T>::NotBootstrapped.into())
```

Reply:
Nice!